examples/kotlin/build.gradle

```diff
@@ -59,18 +59,15 @@ dependencies {
   compile library.java.google_api_services_pubsub
   compile library.java.google_auth_library_credentials
   compile library.java.google_auth_library_oauth2_http
-  compile library.java.google_cloud_datastore_v1_proto_client
   compile library.java.google_http_client
   compile library.java.joda_time
-  compile library.java.proto_google_cloud_datastore_v1
   compile library.java.slf4j_api
-  compile library.java.slf4j_jdk14
+  compile "com.google.guava:guava:25.1-jre"
+  compile "org.jetbrains.kotlin:kotlin-stdlib:1.3.72"
```

Comment:
Declare the Kotlin version (1.3.72) as a constant, since we also use it above. https://github.com/apache/beam/blob/d2e1f693edbd06a54d703e3a27897a0b42319fe8/examples/kotlin/build.gradle#L22